src/debug_menu.h

```diff
@@ -15,6 +16,48 @@ class player;
 namespace debug_menu
 {
 
+static enum debug_menu_index : int {
```

Comment:
As well as this. Also, `static enum` doesn't make any sense, whilst `: int` is redundant. Simply `enum debug_menu_index {` would suffice.

I'd consider splitting this `enum` and giving each of the functions its own chunk of it (along with a `switch-case`). However, there's a minor drawback: this would slightly hinder moving the menu items around, but no big deal.

Reply:
i do not think i want to split them at this point. the enums were mainly added to make the switch statement more manageable, and easier to add more to it. if we want to come back later and redo the switch statement we can.

Reply:
I'm a bit on the fence about splitting them either: we don't want more than one level of nesting, do we? So I'm fine with leaving the switch statement as it is for now.